cmd/aries-agent-mobile/pkg/wrappers/rest/aries.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This is essentially a REST client.

It's a bad idea to add the cross-language barrier (java-go) on top of the REST network call. It's better to just implement a REST client in the Java/Android layer, similar to how aries-js-worker has a REST client version.

Reply:
@llorllale this is related to what myself and @troyronda discussed. how many libraries to have per target and how much of the library we want written outside of go. what gomobile does is to convert golang code into a .so (shared object) which cgo uses to create a .framework or .aar.

if the http client is written outside of our bindings, we would have to import our generated code into a new objectivec/java library. this library will be the final product that exposes both implementations. However with iOS, we will be creating something called an Umbrella Framework which has been [discouraged by Apple](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/CreationGuidelines.html).